packages/client-s3-browser/README.md

```diff
@@ -0,0 +1,135 @@
+# @aws-sdk/client-s3-browser
+
+## Description
+
+Amazon Simple Storage Service service
+
+## Installing
+
+To install the this package using NPM, simply type the following into a terminal window: 
+
+```
+npm install @aws-sdk/client-s3-browser
+```
+
+## Getting Started
+
+### Import
+
+The AWS SDK is modulized by clients and commends in CommonJS modules. To send a request, you only need to import the client(`S3Client`) and the commands you need, for example `PutObjectCommand`:
```

Comment:
There's a typo here, "commends", and a few other style issues in the README.

Those could be addressed in a separate PR.

Reply:
Thanks!